jsonnet/custom.libsonnet

```diff
@@ -79,6 +79,21 @@
               severity: 'critical',
             },
           },
+          {
+            alert: 'highOverallControlPlaneIOWait',
+            expr: |||
+              (sum(irate(node_cpu_seconds_total {mode="iowait"} [2m])) without (cpu)) / count(node_cpu_seconds_total) without (cpu) * 100 > 20
```

Comment:
Hi @openshift/sre-alert-sme folks, we are trying to build an alert to detect disk/network pressure causing CPU latency. Based on your experience with OCP, would 20% of CPU time waiting for IO make sense to start alerting on?

Also what could be a meaningful threshold for a critical alert?